daml-model/daml/Model/LoanConfig.daml

```diff
@@ -36,18 +39,21 @@ template LoanConfig
 
     where
         signatory provider
-        observer accountHolder
+        observer getObserver accountHolder
         key (BalanceKey with iban = account, provider = provider) : BalanceKey
         maintainer key.provider
 
         ensure activeLoan >= 0.0
             && activeLoan <= maxLoanSize
             && balanceLoanThreshold >= 0.0
-            && balanceLoanThreshold <= maxLoanSize
-            && balancePaybackThreshold > maxLoanSize
+            && balancePaybackThreshold >= balanceLoanThreshold
```

Comment:
This is to make sure we don't get into a loan taking <-> payback loop by having the threshold of the payback being lower than the loan taking.

Reply:
Makes sense